packages/xo-web/src/common/intl/messages.js

```diff
@@ -2372,9 +2372,9 @@ const messages = {
     'This will disconnect each selected SR from its host (local SR) or from every hosts of its pool (shared SR).',
   srForgetModalTitle: 'Forget SR',
   srsForgetModalTitle: 'Forget selected SRs',
-  srForgetModalMessage: "Are you sure you want to forget this SR? VDIs on this storage won't be removed.",
+  srForgetModalMessage: "Are you sure you want to forget this SR? You will lose all the metadata for it, meaning all the links between the VDIs (disks) and their respective VMs. This operation cannot be undone.",
   srsForgetModalMessage:
-    "Are you sure you want to forget all the selected SRs? VDIs on these storages won't be removed.",
+  "Are you sure you want to forget {nSrs, number} SR{nSrs, plural, one {} other {s}}? You will lose all the metadata for it, meaning all the links between the VDIs (disks) and their respective VMs. This operation cannot be undone.",
```

Comment:
Maybe the new sentence should be a separate message: it would avoid duplication and it would allow it to be translated since you're not updating the other locales right now.

Reply:
So create two distinct messages ?
first message "Are you sure you want to forget this SR?"
and second message "You will lose all the metadata for it, meaning all the links between the VDIs (disks) and their respective VMs. This operation cannot be undone."

Reply:
Exactly.